CONTRIBUTING.md

```diff
@@ -66,7 +67,7 @@ BUILD_SOX=1 python setup.py develop
 # DEBUG=1 python setup.py develop
 ```
 
-If you built sox, set the `PATH` variable so that the tests properly use the newly built `sox` binary:
+If you built sox, set the `PATH` variable so that the tests properly use the newly built `SoX` binary:
```

Comment:
This is wrong. `sox` is a command line.

Reply:
Thanks for pointing it out. I've fixed it in the latest command,.